unittests/contracts/eosio.mechanics/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/contracts/ DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/contracts/)
```

Comment:
This should do the same thing as the other CMakeLists.txt of eosio.wrap and eosio.token etc